### PR TITLE
Add ABOUT_ME.md file

### PR DESCRIPTION
**Pull request by shaktipn**
I have added the ABOUT_ME.md file which contains a few lines about me.
_Please verify_

### CODE REVIEW

File: exercises/creating-a-pull-request/ABOUT_ME.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# About me
+
+## Shakti Prasad Nanda - name
+
+Hi I am **Shakti** !
```
Comment (sudarshan-dodiya) [inline exercises/creating-a-pull-request/ABOUT_ME.md:5]:
```suggestion
Hi, I am **Shakti**!
```

File: exercises/creating-a-pull-request/ABOUT_ME.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+*I like experimenting stuffs and curious about stuff that people find mundane.*
```
Comment (sudarshan-dodiya) [inline exercises/creating-a-pull-request/ABOUT_ME.md:6]: This line does not make sense to me. If unsure about grammar, please use Grammarly or similar tool to proofread your text.

File: exercises/creating-a-pull-request/ABOUT_ME.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# About me
+
+## Shakti Prasad Nanda - name
```
Comment (sudarshan-dodiya) [inline exercises/creating-a-pull-request/ABOUT_ME.md:3]: Why is `- name` still there?

**Review by sudarshan-dodiya — CHANGES_REQUESTED** [review]
Also, please make sure that your title starts with a capital letter. 
The commit message does not follow the rules of a good commit message: https://coda.io/d/_dHOXTAtFBHO/Creating-Pull-Requests_sulg3#_luNjv

**Comment by shaktipn** [comment]
@sudarshan-dodiya I have modified as per the feedback received. Please review again. Thanks !

File: exercises/creating-a-pull-request/ABOUT_ME.md (hunk 1 above)
Comment (shaktipn) [reply exercises/creating-a-pull-request/ABOUT_ME.md]: I have added the missing _comma_ after Hi.

File: exercises/creating-a-pull-request/ABOUT_ME.md (hunk 2 above)
Comment (shaktipn) [reply exercises/creating-a-pull-request/ABOUT_ME.md]: I have changed that line to say something else.

File: exercises/creating-a-pull-request/ABOUT_ME.md (hunk 3 above)
Comment (shaktipn) [reply exercises/creating-a-pull-request/ABOUT_ME.md]: I have removed it now

**Review by sudarshan-dodiya — APPROVED** [review]

**Pull request merged**